Exiled.API/Features/Map.cs

```diff
@@ -397,6 +397,42 @@ public static void UnlockAllDoors()
                 door.ChangeLock(DoorLockType.None);
         }
 
+        /// <summary>
+        /// Gets an random <see cref="Room"/>.
+        /// </summary>
+        /// <returns><see cref="Room"/> object.</returns>
+        public static Room GetRandomRoom() => Rooms[Random.Range(0, Rooms.Count)];
```

Comment:
Recommend:
`GetRandomRoom(ZoneType type = ZoneType.Unspecified)` to filter by zone.